Dockerfile.ubi

```diff
@@ -37,10 +37,13 @@ ARG STATIC_LINK=no
 RUN make
 
 # hash below referred to latest
-FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:33931dce809712888d1a8061bfa676963f517daca993984afed3251bc1fb5987
+FROM registry.access.redhat.com/ubi8-minimal@sha256:4ed971a5564d2cfedf750793da53ef6f1fdf295263f1cc31d703aa8acf6d02cf
```

Comment:
Can we add this and the `go-toolset` image above to `dependencies.yaml`?

Reply:
good idea, done

Reply:
oh, I haven't added go-toolset. On it.

Reply:
go-toolset added